docs/pages/guides/adopting-prebuild.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Adopting Prebuild
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+There are many advantages to using Prebuild to manage your native projects. This guide will show you how to adopt prebuilding in a project that was bootstrapped with `npx react-native init`.
+
+> **Note:** Adopting prebuild automatically adds support for developing modules with the [Sweet API][expo-modules-core] by linking `expo-modules-core` natively. You can also use any command from Expo CLI `npx expo` in your project.
+
+## Bootstrap a new project
+
+<!-- NOTE: Update the version when we bump support -->
+
+<Terminal cmd={["$ npx react-native init --version 0.69.4"]} />
+
+This will create a new React Native project. Note that not all versions of `react-native` are supported, be sure to use a version of `react-native` that has a corresponding Expo SDK version.
```

Comment:
```suggestion
This will create a new React Native project. Note that [not all versions of `react-native` are explicitly supported](/versions/latest/#each-expo-sdk-version-depends-on-a), be sure to use a version of `react-native` that has a corresponding Expo SDK version.
```